_posts/2018-05-29-mono-on-aix-and-ibm-i.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: blog
+title: Introducing Mono on AIX and IBM i
+author: Calvin Buckley
+tags: [porting]
+---
+
+<!-- Edited by Mark Waterbury -->
+
+[![C# REPL running under IBM i](/images/mono-os400.png)](/images/mono-os400.png)
+
+You might have noticed this in the
+[Mono 5.12 release notes](http://www.mono-project.com/docs/about-mono/releases/5.12.0/),
+Mono now includes support for
+[IBM AIX and IBM i](/docs/about-mono/supported-platforms/aix); two very
+different yet (mostly!) compatible operating systems. This post should serve as
+an introduction to this port.
+
+## What does it take to port Mono?
+
+Porting Mono to a new operating system is not as hard as you might think!
```

Comment:
Use exclamation sparingly!

Reply:
I think I'm subconsciously imitating the enthusiastic style you'd see on [Julia Evans](https://jvns.ca/)' blog; I find it works well for introducing strange concepts to an audience willing to learn.